src/chapters/angular-wk1/angular-1st-steps.rst

```diff
@@ -0,0 +1,236 @@
+Angular First Steps
+====================
+
+The goal here is to create a minimum working webpage template. It will serve as
+a jumping off point for your later exercises and studios, so be sure you are
+logged into your GitHub account.
+
+Starting a New Project
+-----------------------
+
+Navigate to the `StackBlitz homepage <https://stackblitz.com>`__. Click "Start
+New App", then select the "Angular (TypeScript)" option.
+
+.. figure:: ./figures/StackBlitzHome.png
+   :alt: StackBlitz homepage selections.
+
+Examine the files created:
+
+.. figure:: ./figures/NewProjectFiles.png
+   :alt: File tree for new StackBlitz Angular project.
+
+#. The ``src`` folder holds all the files needed to make the template function.
+#. The ``app`` folder holds the instructions for displaying the smaller pieces
+   of a webpage. We will modify these files soon.
+#. ``index.html`` is the highest level for displaying content. Anything added
+   to this HTML file will appear on every page within a website.
+#. ``main.ts`` imports the core methods required to make everything work. It
+   also imports the content from the ``app`` folder.
+#. ``styles.css`` holds the global style settings for the entire website.
+
+Yay! A Webpage!
+----------------
+
+For a better view, go ahead and close the preview panel, then click the button
+to open that preview in a new browser window. You should see some simple
+content.
+
+.. figure:: ./figures/HelloAngular.png
+   :alt: StackBlitz new app default page.
+
+Congratulations! You have a functioning webpage. Feel free to play around a
+little bit before continuing, and do not worry about breaking anything. If
+necessary, you can always start another new project.
+
+.. _try-it-StackBlitz-intro:
+
+.. admonition:: Try It!
+
+   #. Add a nose to the emoticon.
+   #. Find where "Angular" is assigned to the heading, and then replace it
+      with your name.
+   #. Change the color for the *Start editing...* sentence.
+
+   You will need to refresh the preview page after each attempt to see if any
+   changes occur.
+
+Now let's take a look at the different project files.
+
+What To Ignore
+^^^^^^^^^^^^^^^
+
+For every new project, StackBlitz includes the practice file
+``hello.component.ts``. It plays a role in generating the "Hello Angular"
+heading, but it does not do anything other than that. For our discussion here,
+we will ignore the file.
+
+.. admonition:: Warning
```

Comment:
I thought about providing guidance for removing the ``hello`` component, but those instructions got bogged down in "right-click-here", etc.

I was also worried that if the instructions were not followed exactly right, then a student's project would not get off the ground.

Reply:
With that information, I would agree. Let's have the student focus on creating, instead of deleting. I wouldn't change anything then. You can mark this comment as resolved if you agree nothing needs to be changed.

Reply:
OK.